### PR TITLE
Adding Lint

### PR DESCRIPTION
**Pull request by yashsharma81**
Added <issue id="NoHardKeywords" severity="fatal"/>

## Purpose / Description
_Describe the problem or feature and motivation_

## Fixes
Fixes _Link to the issues._

## Approach
_How does this change address the problem?_

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: lint-release.xml [hunk 1]
```diff
@@ -180,6 +180,7 @@
     <issue id="StringShouldBeInt" severity="ignore" />
     <issue id="NewerVersionAvailable" severity="ignore" />
     <issue id="MinSdkTooLow" severity="ignore" />
+    <issue id="NoHardKeywords" severity="fatal"/>
```
Comment (david-allison) [inline lint-release.xml:183]: This didn't work, we should have lint failures which need to be fixed.

**Review by david-allison — CHANGES_REQUESTED** [review]
We should have lint failures as we're using `when` in tests.

**Comment by yashsharma81** [comment]
@david-allison ok

**Comment by justdvnsh** [comment]
@yashsharma81 I'd recommend adding more description to the PR. I could not understand anything without seeing the commits. Adding description to the PR according to the given template, would make it easier for everyone to read and get better understanding of the changes.

**Comment by DeepanshuPratik** [comment]
@yashsharma81 I'd recommend adding more description to the PR and also try to have a practice of commenting on the change you did in the codebase. It would be really helpful for others to understand and fix issues in the future.

**Pull request closed without merging**